qchem/tests/test_convert_observable.py

```diff
@@ -1,556 +0,0 @@
-import os
```

Comment:
Out of curiosity, were these tests moved else where, or was the underlying code deprecated?

Reply:
They were moved to `test_convert.py` in a previous PR.